guides/common/assembly_host-management-and-monitoring-using-cockpit.adoc

```diff
@@ -6,4 +6,4 @@ include::modules/proc_managing-and-monitoring-hosts-using-cockpit.adoc[leveloffs
 
 ifndef::foreman-deb[]
 include::modules/proc_disabling-cockpit-on-project.adoc[leveloffset=+1]
-endif::[]
\ No newline at end of file
+endif::[]
```

Comment:
Unsure what's going on here. Maybe file ending things?

Reply:
Yeah when I added the module originally in #1989 I did not include the newline on the last line. this is visible when running `git show` with the commit hash. Since it is a Unix practice I just added the missing newline in this commit.